docker/launch_backend_service.sh

```
@@ -68,7 +68,10 @@ STOP=false
 PIDS=()
 
 # Set the path to the NLTK data directory
-export NLTK_DATA="./nltk_data"
+# download_deps.py downloads NLTK data into ragflow_deps/nltk_data (see
+# ragflow_deps/download_deps.py); point NLTK_DATA there directly instead of a
+# stale top-level ./nltk_data so this matches what's actually populated.
+export NLTK_DATA="$(pwd)/ragflow_deps/nltk_data"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)